app/controllers/users_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
     lists_of_users
   end
 
+  # rubocop:disable Style/GuardClause
   def lists_of_users
```

Comment:
**OPTIONAL**
Can be simplified by adding some methods taking advantage of ActiveRecord in the `User` model.

Read the other comments to know more about it.